internal/web/validators/validatorurl.go

```diff
@@ -1,37 +1,39 @@
-package netaddr
+package validators
 
 import (
-	"log"
 	"net"
 	"net/url"
 	"strings"
+
+	"go.uber.org/zap"
 )
 
 // Parse server and url address
-func CheckAddress(address string) (host string, port string) {
-	log.Println("Parse address: ", address)
+func CheckURL(address string) (host string, port string) {
+
+	zap.S().Infoln("Parse address: ", address)
 	link, err := url.Parse(strings.TrimSpace(address))
 	if err != nil {
-		log.Println("Error parsing url: ", err, " return def localhost:8080")
+		zap.S().Infoln("Error parsing url: ", err, " return def localhost:8080")
 		return "localhost", "8080"
 	}
 
 	//check shema
 	if link.Scheme != "http" {
-		log.Println("Shema not found, use http")
+		zap.S().Infoln("Shema not found, use http")
 		address = "http://" + address
 	}
 
 	link, err = url.Parse(strings.TrimSpace(address))
 	if err != nil {
-		log.Println("Error parsing url whis shema: ", err, " return def localhost:8080")
+		zap.S().Infoln("Error parsing url whis shema: ", err, " return def localhost:8080")
 		return "localhost", "8080"
 	}
 
-	log.Println("Split address: ", link)
+	zap.S().Infoln("Split address: ", link)
 	host, port, err2 := net.SplitHostPort(strings.TrimSpace(link.Host))
```

Comment:
```suggestion
	host, port, err := net.SplitHostPort(strings.TrimSpace(link.Host))
```
никаких err2 быть не должно - ошибка обрабатывается сразу - либо возвращаем либо нам она уже не нужна и переменную можно перезаписать

Reply:
Запомнил